packages/angular/src/builders/webpack-browser/webpack-browser.impl.ts

```diff
@@ -5,41 +5,45 @@ import {
   createBuilder,
   targetFromTargetString,
 } from '@angular-devkit/architect';
+import { executeBrowserBuilder } from '@angular-devkit/build-angular';
 import { JsonObject } from '@angular-devkit/core';
 import { from, Observable, of } from 'rxjs';
 import {
   calculateProjectDependencies,
   checkDependentProjectsHaveBeenBuilt,
   createTmpTsConfig,
 } from '@nrwl/workspace/src/utils/buildable-libs-utils';
-import { join } from 'path';
+import { join, normalize } from 'path';
 import { createProjectGraph } from '@nrwl/workspace/src/core/project-graph';
 import { Schema } from '@angular-devkit/build-angular/src/browser/schema';
 import { switchMap } from 'rxjs/operators';
+import { existsSync } from 'fs';
+import { merge } from 'webpack-merge';
 
 type BrowserBuilderSchema = Schema &
   JsonObject & {
     buildTarget: string;
+    customWebpackConfig: {
+      path: string;
+    };
+    targetBuilder: string;
   };
 
 function buildApp(
   options: BrowserBuilderSchema,
   context: BuilderContext
 ): Promise<BuilderRun> {
-  const { buildTarget, ...delegateOptions } = options;
-
-  // delete targetBuilder from options before passing into scheduleBuilder
-  const targetBuilder =
-    (delegateOptions.targetBuilder as string) || buildTarget;
-  delete delegateOptions.targetBuilder;
+  const { buildTarget, targetBuilder, ...delegateOptions } = options;
 
   if (buildTarget) {
-    const target = targetFromTargetString(targetBuilder);
+    const builder = (targetBuilder as string) || buildTarget;
```

Comment:
Casting shouldn't be needed now that `targetBuilder` was added to `BrowserBuilderSchema ` with the correct type.

Reply:
https://github.com/nrwl/nx/pull/4903/files#diff-ca1cfeffd2ac9b79db54ee8a3e0aabe8091a22d7d77a09ea271d9503be603f64

That was required :/ some reason the type wasn't being picked up correctly

Reply:
It was not picked up before because `targetBuilder` was not part of the `BrowserBuilderSchema` type definition. This PR [is adding that property to the type](https://github.com/nrwl/nx/pull/5609/files#diff-ca1cfeffd2ac9b79db54ee8a3e0aabe8091a22d7d77a09ea271d9503be603f64R29), so the type should be picked up correctly now.

Reply:
Yep! I added the type to the object afterwards. Can remove this casting

Reply:
Sweet, sorry i must of missed it